src/node_http2.cc

```diff
@@ -14,17 +14,19 @@ using v8::Context;
 using v8::Float64Array;
 using v8::Function;
 using v8::Integer;
+using v8::ObjectTemplate;
 using v8::String;
 using v8::Uint32;
 using v8::Uint32Array;
 using v8::Undefined;
 
 namespace http2 {
 
-Nghttp2Session::Callbacks Nghttp2Session::callback_struct_saved[2] = {
+Http2Session::Callbacks Http2Session::callback_struct_saved[2] = {
```

Comment:
While you’re at it, could you make these `const`? Otherwise it’s technically global mutable state…